tests/data_context/test_cloud_migrator_configuration_bundle.py

```diff
@@ -1,246 +1,265 @@
+            anonymous_usage_statistics = None
+        else:
+            anonymous_usage_statistics = AnonymizedUsageStatisticsConfig(
+                enabled=self._anonymous_usage_stats_enabled
+            )
+
+        config = DataContextConfig(
+            anonymous_usage_statistics=anonymous_usage_statistics
         )
-    )
-    assert list_of_dicts_equal(config_bundle._profilers, [roundtripped_profiler_config])
+        return EphemeralDataContextVariables(config=config)
+
+    @property
+    def _datasource_store(self):
+        return StubDatasourceStore()
+
+    @property
+    def datasources(self) -> Dict[str, Union[LegacyDatasource, BaseDatasource]]:
+        # Datasource is a dummy since we just want the DatasourceConfig from the store, not an
+        # actual initialized datasource.
+        return {"my_datasource": DummyDatasource()}
+
+    @property
+    def checkpoint_store(self) -> StubCheckpointStore:
+        return StubCheckpointStore()
+
+    @property
+    def validations_store(self) -> StubValidationsStore:
+        return StubValidationsStore()
+
+    def list_expectation_suite_names(self) -> List[str]:
+        return ["my_suite"]
+
+    def get_expectation_suite(self, name: str) -> ExpectationSuite:
+        return ExpectationSuite(expectation_suite_name=name)
+
+    def list_checkpoints(self) -> List[str]:
+        return ["my_checkpoint"]
+
+    def list_profilers(self) -> List[str]:
+        return ["my_profiler"]
+
+    def get_profiler(self, name: str) -> RuleBasedProfiler:
+        return RuleBasedProfiler(name, config_version=1.0, rules={})
+
+
+@pytest.fixture
+def stub_base_data_context() -> StubBaseDataContext:
+    return StubBaseDataContext()
+
+
+@pytest.fixture
+def stub_base_data_context_anonymous_usage_stats_present_but_disabled() -> StubBaseDataContext:
+    return StubBaseDataContext(anonymous_usage_stats_enabled=False)
+
 
-    assert config_bundle._validation_results == [
-        in_memory_runtime_context_with_configs_in_stores.validation_result
-    ]
+@pytest.fixture
+def stub_base_data_context_no_anonymous_usage_stats() -> StubBaseDataContext:
+    return StubBaseDataContext(anonymous_usage_stats_is_none=True)
+
+
+class TestConfigurationBundleCreate:
+    @pytest.mark.unit
+    def test_configuration_bundle_created(
+        self,
+        stub_base_data_context: StubBaseDataContext,
+    ):
+        """What does this test and why?
+
+        Make sure the configuration bundle is created successfully from a data context.
+        """
+
+        context: BaseDataContext = stub_base_data_context
+
+        config_bundle = ConfigurationBundle(context)
+
+        assert config_bundle.is_usage_stats_enabled()
+        assert config_bundle._data_context_variables is not None
+        assert len(config_bundle._expectation_suites) == 1
+        assert len(config_bundle._checkpoints) == 1
+        assert len(config_bundle._profilers) == 1
+        assert len(config_bundle._validation_results) == 1
```

Comment:
I'm adding properties for all these attrs as they need to be accessed for printing logic in the migrator

Reply:
Added properties! (and modified the schema and this test)